### PR TITLE
XW-204 Unwrap Wikia widgets

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/XW-204

### CODE REVIEW

**Comment by hakubo** [comment]
LGTM

File: front/styles/main/module/_wikia-widget.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+iframe[data-wikia-widget] {
+	width: 100%;
+}
```
Comment (bognix) [inline front/styles/main/module/_wikia-widget.scss]: is this needed if we handle all iframes somewhere else?

Comment (rogatty) [inline front/styles/main/module/_wikia-widget.scss]: We have
```
iframe {
	max-width: 100%;
}
```
in `_media.scss` but I don't want to put this rule there. The one above doesn't touch all iframes but only our widgets so I can't introduce any regression - I don't like introducing them ;)

**Comment by bognix** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**